### PR TITLE
chore(zappr): require approval of Norris+UX to merge

### LINKED ISSUE #273: Change approval process
As agreed in today's meeting between UX + Norris frontend devs we will switch to a new approval system where two approvals, one from UX and one from an engineer are needed in order to be able to merge a PR.

We need to adjust both the repository configuration and the Zappr config file in order to allow this.

### PR DESCRIPTION
**Pull request by dami-gg**
Closes #273

### CODE REVIEW

File: .zappr.yaml [hunk 1]
```diff
@@ -1,15 +1,20 @@
...
-    orgs:
-      # a member of norris organization
-      # (any org in here counts)
+  groups:
+   # there must be at least a technical approval from a member of team Norris
+   technical:
+    minimum: 1
+    from:
+     orgs:
       - norris
```
Comment (cjhowald) [inline .zappr.yaml]: Aren't these GHE orgs? Will they work as expected in GH?

Comment (dami-gg) [inline .zappr.yaml]: They're also GH, I created the UX one this time (https://github.com/orgs/zalando/teams/ux-brand-solutions) and the Norris one somewhen in the past, and it was actually already in use (https://github.com/orgs/zalando/teams/norris)

Comment (gregdeane) [inline .zappr.yaml]: Do you still have to add ux-brand-solutions to the "Collaborators" section under "Settings"?

Comment (dami-gg) [inline .zappr.yaml]: Good point, I forgot that.

image

Moreover, I think I can remove all of us and UX from collaborators now right?

Comment (gregdeane) [inline .zappr.yaml]: I would think so.

Comment (gregdeane) [inline .zappr.yaml]: One other thing... I don't see the zappr structure that you've included in any of the following examples:

* https://github.com/zalando/zappr/blob/master/config/app-opensource.yaml
* https://github.com/zalando/zappr/blob/master/config/app-enterprise.yaml
* https://gist.github.bus.zalan.do/tyang/37575c0177beb06cdab4ced60e4bd389

How can you tell that the zappr structure you've added is correct?

Comment (dami-gg) [inline .zappr.yaml]: done

Comment (dami-gg) [inline .zappr.yaml]: it's not exactly like that in the docs but talked to @mfellner and he confirmed that syntax is right

**Comment by cjhowald** [comment]
👍

**Comment by dami-gg** [comment]
Let's wait until we merge all the tasks today in order to speed it up and then we enable it.

**Pull request merged**